python_modules/dagster/dagster/_core/pipes/utils.py

```diff
@@ -444,19 +444,9 @@ def _logs_thread(
         is_session_closed: Event,
         messages_thread: Thread,
     ) -> None:
-        # Start once we have received the opened message from the external process
-        while True:
-            if self.opened_payload is not None:
-                break
-            # We never received the `opened` message and never will, so don't try to start the log
-            # reader threads.
-            elif not messages_thread.is_alive():
-                return
-            time.sleep(DEFAULT_SLEEP_INTERVAL)
-
-        # Logs are started with a merge of the params generated by the message reader and the opened
-        # payload.
-        log_params = {**params, **self.opened_payload}
+        # Logs are started with a merge of the params generated by the message reader, the opened
+        # payload, and params from the state
+        log_params = {**params, **(self.opened_payload or {})}
```

Comment:
These are immediately set at the top of the loop so why set them here at all? They aren't used in a call before the loop starts.